addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -69,7 +69,8 @@ $o-btn-outline-border-width-defaults: () !default;
         } @else {
             @extend .btn-fill-#{$color};
             @if index($o-btn-flat-defaults, $color) {
-                @extend .btn.flat;
+                // TODO In master: move definition to web_editor
```

Comment:
@nboulif nboulif
This is the master PR.
So why you don't move definition to web_editor ?

Reply:
This is what I thought at first, but since a deeper investigation must be done and might delay this fix, I preferred to let this one go and create a ticket for master to investigate what's the best way to handle the refractor.
What do you think?

Reply:
Ok, but in this case the TODO message is not correct.
But I'm ok with your proposition.

FYI: as it's a forward port you can r+ your self